sched/environ/env_dup.c

```diff
@@ -64,8 +64,8 @@
 int env_dup(FAR struct task_group_s *group)
 {
   FAR struct tcb_s *ptcb = this_task();
-  FAR char *envp = NULL;
-  size_t envlen;
+  FAR char **envp = NULL;
+  size_t envlen = 0;
```

Comment:
probably renaming `envlen` -> `envc` can be a good option.

Reply:
Done.